components/mintmaker/base/renovate-config.yaml

```diff
@@ -7,19 +7,13 @@ data:
   renovate.json: |
```

Comment:
Will there be another PR to enable only the three managers later, after we change the Renovate image to our custom image that supports the RPM manager?

Reply:
No, I just forgot :) I've updated this PR to have only those 3 managers.
BTW I didn't add a preset on purpose, I see our (rebuilds guild) preset might not be ok with this conf. So let's just skip it for now, I don't really see the benefit at this point. We can see if we need a preset in the future.